tests/conformance/README.md

```diff
@@ -2,81 +2,30 @@
+The conformance test for buildah is used to verify the images built with Buildah are equivalent to those built by Docker.  It does this by building an image using the version of buildah library that's being tested, building what should be the same image using `docker build`, and comparing them.
 
 ## Installing dependencies
 
-The dependencies for conformance testing include two parts:
-* Binary required by the conformance test suite:
+The additional dependencies for conformance testing are:
   * docker
-  * podman
-  * container-diff
-* Binary required to run the tests:
-  * ginkgo
-
-### Install Buildah
-
-Install Buildah using dnf, yum or apt-get, based on your distribution.  Buildah can also be cloned and installed from [GitHub](https://github.com/containers/buildah/blob/master/install.md).
 
 ### Install Docker CE
 
-Conformance tests use Docker CE to check the images built with Buildah. Install Docker CE with dnf, yum or apt-get, based on your distribution and verify that the Docker service is started as the default. In Fedora, RHEL and CentOS Docker rather than Docker CE may be installed by default, please verify that you install the correct variant of Docker.
-
-### Install Podman
-
-[Podman](https://github.com/containers/podman) is used to push images built with Buildah to the docker daemon. It can be installed with dnf or yum in Fedora, RHEL and CentOS, it also can be installed from source code. If you want to install Podman from source code, please follow the [libpod Installation Instructions](https://github.com/containers/podman/blob/master/install.md).
-
-### Install container-diff
-
-[container-diff](https://github.com/GoogleContainerTools/container-diff) is used for check images file system from Buildah and Docker. It can be installed with the following command:
-
-```
-curl -LO https://storage.googleapis.com/container-diff/latest/container-diff-linux-amd64 && chmod +x container-diff-linux-amd64 && mkdir -p $HOME/bin && export PATH=$PATH:$HOME/bin && mv container-diff-linux-amd64 $HOME/bin/container-diff
-```
-
-### Install ginkgo and gomega
-
-We already have vendored ginkgo and gomega into the conformance tests, so if you want to just run the tests with "go test", you can skip this step.
-Ginkgo is tested with Go 1.6+, please make sure your golang version meet the required version.
-```
-go get -u github.com/onsi/ginkgo/ginkgo  # installs the ginkgo CLI
-go get -u github.com/onsi/gomega/...     # fetches the matcher library
-export PATH=$PATH:$GOPATH/bin
-```
+Conformance tests use Docker CE to build images to be compared with images built with Buildah.  Install Docker CE with dnf, yum or apt-get, based on your distribution and verify that the `docker` service is started.  In Fedora, RHEL and CentOS `docker` or `moby-engine` rather than Docker CE may be installed by default.  In Debian or Ubuntu you may instead have the `docker.io` package.  Please verify that you install a recent version Docker.
```

Comment:
"recent" is meaningless in static documents. If there is a hard dependency, could you state it? If you mean 'the most recent version of Docker available to you', please state it that way or similarly. If you mean something else, please clarify.

Reply:
Right, it's also badly phrased.

Reply:
Replaced that sentence with `Please verify that you install at least version 19.03.`